🐛 BUG: C3 fails to bootstrap a new Remix project

### Which Cloudflare product(s) does this pertain to?

C3 (npm create cloudflare)

### What version(s) of the tool(s) are you using?

2.34.0[C3]

### What version of Node are you using?

22.12.0

### What operating system and version are you using?

MacOS [15.1.1]

### Describe the Bug

### Observed behavior
When scaffolding a new Remix project with the command `npm create cloudflare@latest my-remix-app -- --framework=remix --experimental`, I run into an error (error log shared below).

This is an error with `create-remix@2.15.0`.  Here's the link to the fixed PR: https://github.com/remix-run/remix/pull/10300

We need to update the create-remix version.

### Expected behavior
The command should scaffold a new Remix project without any error.

### Steps to reproduce
In a terminal, try executing the command: `npm create cloudflare@latest my-remix-app -- --framework=remix --experimental`.


### Please provide a link to a minimal reproduction

_No response_

### Please provide any relevant error logs

```
Error: Cannot find module 'fs-extra'
Require stack:
- /Users/harshil/.npm/_npx/2e924abdc2735788/node_modules/create-remix/dist/index.js
- /Users/harshil/.npm/_npx/2e924abdc2735788/node_modules/create-remix/dist/cli.js
    at Function._resolveFilename (node:internal/modules/cjs/loader:1249:15)
    at Function._load (node:internal/modules/cjs/loader:1075:27)
    at TracingChannel.traceSync (node:diagnostics_channel:322:14)
    at wrapModuleLoad (node:internal/modules/cjs/loader:219:24)
    at Module.require (node:internal/modules/cjs/loader:1340:12)
    at require (node:internal/modules/helpers:138:16)
    at Object.<anonymous> (/Users/harshil/.npm/_npx/2e924abdc2735788/node_modules/create-remix/dist/index.js:19:11)
    at Module._compile (node:internal/modules/cjs/loader:1565:14)
    at Object..js (node:internal/modules/cjs/loader:1708:10)
    at Module.load (node:internal/modules/cjs/loader:1318:32) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    '/Users/harshil/.npm/_npx/2e924abdc2735788/node_modules/create-remix/dist/index.js',
    '/Users/harshil/.npm/_npx/2e924abdc2735788/node_modules/create-remix/dist/cli.js'
  ]
}

Node.js v22.12.0
╰  ERROR  Error
```

--- a/.changeset/new-shoes-deny.md
+++ b/.changeset/new-shoes-deny.md
@@ -1,0 +1,5 @@
+---
+"create-cloudflare": patch
+---
+
+fix: update create-remix to fix creating a Remix project failing with "Cannot find module 'fs-extra'"

--- a/packages/create-cloudflare/src/frameworks/package.json
+++ b/packages/create-cloudflare/src/frameworks/package.json
@@ -13,7 +13,7 @@
 		"create-next-app": "14.2.5",
 		"create-qwik": "1.11.0",
 		"create-vite": "6.0.1",
-		"create-remix": "2.15.0",
+		"create-remix": "2.15.1",
 		"create-solid": "0.5.14",
 		"create-vue": "3.12.1",
 		"gatsby": "5.14.0",

